packages/stripe-integration/src/stripev3/stripev3-payment-strategy.ts

```diff
@@ -714,7 +716,9 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
             try {
                 return await this.paymentIntegrationService.submitPayment(paymentPayload);
             } catch (error) {
-                throw this.handleEmptyPaymentIntentError(error, result?.error);
+                if (error instanceof Error || isRequestError(error)) {
```

Comment:
The same as above

Reply:
At the time of the comment, it was already removed.
No `error instanceof Error || isRequestError(error)` in the repo now.